cpp/daal/include/algorithms/k_nearest_neighbors/bf_knn_classification_model.h

```diff
@@ -52,6 +52,25 @@ enum DataUseInModel
     doNotUse = 0, /*!< The input data and labels will not be the component of the trained kNN model */
     doUse    = 1  /*!< The input data and labels will be the component of the trained kNN model */
 };
+/**
+ * <a name="DAAL-ENUM-ALGORITHMS__BF_KNN_CLASSIFICATION__RESULTTOCOMPUTEID"></a>
+ * Available identifiers to specify the result to compute
+ */
+enum ResultToComputeId
+{
+    computeClassLabels         = 0x00000001ULL,
+    computeIndicesOfNeightbors = 0x00000002ULL,
+    computeDistances           = 0x00000004ULL
+};
+/**
+ * <a name="DAAL-ENUM-ALGORITHMS__BF_KNN_CLASSIFICATION__VOTEWEIGHTS"></a>
+ * \brief Weight function used in prediction voting
+ */
+enum VoteWeights
+{
+    voteUniform  = 0,
+    voteDistance = 1
+};
```

Comment:
Add doxygen. Also, does it makes sense to generalize for bf & kd-tree?

Reply:
Added same as kd-tree